lib/awaitableSender.ts

```diff
@@ -170,17 +185,25 @@ export class AwaitableSender extends BaseSender {
    * @param {number} [format] The message format. Specify this if a message with custom format needs
    * to be sent. `0` implies the standard AMQP 1.0 defined format. If no value is provided, then the
    * given message is assumed to be of type Message interface and encoded appropriately.
-   * @param {number} [timeoutInSeconds] If provided, this timeout overrides the `sendTimeoutInSeconds`
-   * that is set when the `AwaitableSender` is created. This timeout represents the duration in which
-   * the promise to send the message should complete (resolve/reject). If not, the Promise will be
-   * rejected after timeout.
+   * @param {AwaitableSendOptions} [options] Options to configure the timeout and cancellation for
+   * the send operation.
    * @returns {Promise<Delivery>} Promise<Delivery> The delivery information about the sent message.
    */
-  send(msg: Message | Buffer, tag?: Buffer | string, format?: number, timeoutInSeconds?: number): Promise<Delivery> {
+  send(msg: Message | Buffer, tag?: Buffer | string, format?: number, options?: AwaitableSendOptions): Promise<Delivery> {
```

Comment:
This is not considered a breaking change?

Reply:
timeoutInSeconds was introduced in https://github.com/amqp/rhea-promise/pull/53 which has not been released yet. So, no breaking changes